tutorials/dataframe/tdf012_using_jit.C

```diff
@@ -0,0 +1,63 @@
+/// \file
+/// \ingroup tutorial_tdataframe
+/// \notebook -nodraw
+///
+/// This tutorial illustrates how to save some typing when using TDataFrame
+/// by invoking functions that perform jit-compiling at runtime.
+///
+/// \macro_code
+///
+/// \date October 2017
+/// \author Guilherme Amadio
+
+#include "TRandom.h"
+#include "ROOT/TDataFrame.hxx"
+
+void tdf012_using_jit()
+{
+   // We will inefficiently calculate an approximation of pi by generating
+   // some data and doing very simple filtering and analysis on it
+
+   // We start by creating an empty dataframe where we will insert 10 million
+   // random points in a square of side 2.0 (that is, with an inscribed circle
+   // of radius 1.0)
+
+   size_t npoints = 10000000;
+   ROOT::Experimental::TDataFrame tdf(npoints);
+
+   // Make generators known to interpreter
+   gInterpreter->ProcessLine("TRandom rx, ry;");
+
+   // Use different seeds for independent streams
+   gInterpreter->ProcessLine("rx.SetSeed(1);");
+   gInterpreter->ProcessLine("ry.SetSeed(2);");
```

Comment:
We have to define the generator out of the lambda, otherwise, we'd be setting the seed every time and getting only the first number from the sequence.